### PR TITLE
Add OpenSearch style guide announcement

### PR DESCRIPTION
**Pull request by Naarcha-AWS**
Signed-off-by: Naarcha-AWS <naarcha@amazon.com>

### Description
Adds an announcement blog post for the OpenSearch style guide on behalf of Nate Bower. Also adds Nate's photo and author profile
 

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _authors/nbower.markdown [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: Nate Bower
+short_name: nbower
+photo: '/assets/media/authors/nbower.jpeg'
+twitter: TheAndrewHopp
```
Comment (natebower) [inline _authors/nbower.markdown:5]: I don't have a Twitter, so we can remove this.

File: _authors/nbower.markdown [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+---
+
+Andrew Hopp is the Senior Technical Product Manager for OpenSearch and OpenSearch Dashboards.
```
Comment (natebower) [inline _authors/nbower.markdown:8]: Nate Bower is the Senior Technical Editor for OpenSearch.

**Review by natebower — COMMENTED** [review]
@Naarcha-AWS Two changes, and it should be good to go. Thanks!

**Review by elfisher — APPROVED** [review]
LGTM

**Review by hdhalter — APPROVED** [review]
Looks great!

File: _posts/2022-06-29-announcing-style-guide.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+As part of our focus on producing exceptional content, we are developing quality assurance processes, resources, and initiatives to help us standardize our approach to creating bar-raising OpenSearch content. Our style guidelines represent one of the mechanisms our team will use to achieve this. Check out the style guidelines, and feel free to let us know what you think.
+
+OpenSearch values transparency, and we welcome community contributions to our style guidelines. To access our documentation, see https://opensearch.org/docs/latest. If you want to contribute to the OpenSearch documentation, see the [CONTRIBUTING.md](https://github.com/opensearch-project/documentation-website/blob/main/CONTRIBUTING.md) file, which covers how to open an issue or create a pull request on GitHub. We look forward to collaborating with you!
```
Comment (krisfreedain) [inline _posts/2022-06-29-announcing-style-guide.md:17]: "see https://opensearch.org/docs/latest."
should turn this into a live link

**Review by krisfreedain — CHANGES_REQUESTED** [review]
small change

also, should consider linking to your GitHub account in your profile so people can follow up if needed.

File: _posts/2022-06-29-announcing-style-guide.md (hunk 3 above)
Comment (natebower) [reply _posts/2022-06-29-announcing-style-guide.md:17]: Thanks Kris! @Naarcha-AWS I like both of Kris's suggestions.

**Comment by Naarcha-AWS** [comment]
@krisfreedain: Ready for re-review.

**Comment by krisfreedain** [comment]
> @krisfreedain: Ready for re-review.

not seeing either update - can you look again?

**Comment by Naarcha-AWS** [comment]
There you go @krisfreedain

**Review by krisfreedain — APPROVED** [review]
looks good

**Pull request merged**